brainframe_qt/util/licensing/base.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass
+from enum import Enum, auto
+from typing import Optional
+
+import pendulum
+from brainframe.api import bf_codecs
+
+
+@dataclass
+class LicenseInfo:
+    state: "LicenseState"
+    expiration_date: Optional[pendulum.DateTime]
+    terms: bf_codecs.LicenseTerms  # TODO: Convert to core-specific object
```

Comment:
I think you might have done a Find/Replace for "domain" to "core". Earlier there was a commit that changed a comment from "domain" to "core" in a completely different context :thinking:

Reply:
Oh lol, I renamed a _file_ from `domain.py` to `core.py`. Will fix